IFC 4.3/SpanAnnotation_1/UT_SpanAnnotation_1.ifc

```diff
@@ -0,0 +1,74 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition [Ifc4X3NotAssigned]'),'2;1');
+FILE_NAME(
+/* name */ 'UT_SpanAnnotation_1.ifc',
+/* time_stamp */ '2020-12-02T18:59:05',
+/* author */ ('Sebastian Esser'),
+/* organization */ ('Unknown'),
+/* preprocessor_version */ 'GeometryGymIFC v0.1.4.0 by Geometry Gym Pty Ltd built 2020-11-12T09:32:58',
+/* originating_system */ 'ConsoleCreateSpanAnnotation v1.0.0.0',
+/* authorization */ 'None');
+
+FILE_SCHEMA (('IFC4X3_RC2'));
+ENDSEC;
+
+DATA;
+#1= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#2= IFCAXIS2PLACEMENT3D(#1,$,$);
+#3= IFCLOCALPLACEMENT($,#2);
+#4= IFCSITE('1LZp6F9Ef30QB88kwuTkeU',$,'SiteA',$,$,#3,$,$,$,$,$,$,$,$);
+#5= IFCPROJECT('1rDHlzAEnDIf54Uz6mzze6',$,'SampleProject with a span annotation',$,$,$,$,(#22),#6);
+#6= IFCUNITASSIGNMENT((#7,#8,#9,#10,#11));
```

Comment:
A linear velocity unit should be nominated, the toolkit automated units doesn't include this.  I can help with how to adapt code to achieve this.

Reply:
could you review aeaa95d please? I've added the linearVelocity entity to the project unit assignment. not sure if this is enough or if I should also stage the base units there.